client/blocks/reader-full-post/_content.scss

```
@@ -241,6 +241,35 @@
 		margin: auto;
 	}
 
+	.wp-block-buttons {
+		margin: 0px;
```

```suggestion
		margin: 0;
```

Just a nit - zero doesn't need units :)